pkg/module/metrics/latency.go

```diff
@@ -11,7 +11,6 @@ import (
 	"sync"
 	"time"
 
-	"github.com/cilium/cilium/api/v1/flow"
```

Comment:
I believe there are other references to `flow`, example - https://github.com/microsoft/retina/blob/96ff934a608fcce848d66c31ad8e54afbed9e00e/pkg/module/metrics/latency.go#L60